config.json

```diff
@@ -29,1615 +29,1615 @@
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.mlkit",
         "artifactId": "pose-detection-common",
         "version": "17.0.0",
-        "nugetVersion": "117.0.0.7",
+        "nugetVersion": "117.0.0.8",
         "nugetId": "Xamarin.Google.MLKit.PoseDetection.Common",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.mlkit",
         "artifactId": "smart-reply",
         "version": "17.0.2",
-        "nugetVersion": "117.0.2",
+        "nugetVersion": "117.0.2.1",
         "nugetId": "Xamarin.Google.MLKit.SmartReply",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.mlkit",
         "artifactId": "smart-reply-common",
         "version": "16.1.0",
-        "nugetVersion": "116.1.0",
+        "nugetVersion": "116.1.0.1",
         "nugetId": "Xamarin.Google.MLKit.SmartReply.Common",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.mlkit",
         "artifactId": "translate",
         "version": "17.0.1",
-        "nugetVersion": "117.0.1",
+        "nugetVersion": "117.0.1.1",
         "nugetId": "Xamarin.Google.MLKit.Translate",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.mlkit",
         "artifactId": "vision-common",
         "version": "17.3.0",
-        "nugetVersion": "117.3.0",
+        "nugetVersion": "117.3.0.1",
         "nugetId": "Xamarin.Google.MLKit.Vision.Common",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.mlkit",
         "artifactId": "vision-interfaces",
         "version": "16.1.0",
-        "nugetVersion": "116.1.0",
+        "nugetVersion": "116.1.0.1",
         "nugetId": "Xamarin.Google.MLKit.Vision.Interfaces",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.mlkit",
         "artifactId": "vision-internal-vkp",
         "version": "18.2.2",
-        "nugetVersion": "118.2.2.2",
+        "nugetVersion": "118.2.2.3",
         "nugetId": "Xamarin.Google.MLKit.Vision.Internal.Vkp",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.odml",
         "artifactId": "image",
         "version": "1.0.0-beta1",
-        "nugetVersion": "1.0.0.2-beta1",
+        "nugetVersion": "1.0.0.3-beta1",
         "nugetId": "Xamarin.Google.Android.ODML.Image",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.libraries.places",
         "artifactId": "places",
         "version": "2.2.0",
-        "nugetVersion": "2.2.0",
+        "nugetVersion": "2.2.0.1",
         "nugetId": "Xamarin.GoogleAndroid.Libraries.Places",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.libraries.places",
         "artifactId": "places-compat",
-        "version": "2.1.0",
-        "nugetVersion": "2.1.0",
+        "version": "2.2.0",
```

Comment:
This is not just a patch bump, this is a new Java version.

Reply:
I had to manually bump some (remember order of last updates FB was 1st but we merged it as last).
Will fix right now.

Reply:
fixed 2.2.0.1